### ISSUE TITLE
Question: Rename color to tint?

### ISSUE THREAD

**Issue by benjaminbojko** [bluecadet/Cinder-BluecadetViews#5]
`BaseView` has a `mColor` property and `getColor()`/`setColor()` accessors. I feel like that name becomes a little confusing once we start subclassing. For example, `TextView` has both `mTextColor` and `mColor`. Both have an effect on how text color is rendered, but `mColor` takes precedence.

Essentially `mColor` simply gets multiplied with whatever happens in `BaseView::draw()`. So if the `TextView` renders white text and `mColor` is set to red, the text will become red because:

``` c++
r = 1.0 * 1.0; // 1
g = 0 * 0; // 0
b = 0 * 0; // 0
```

If `mColor` is blue and `mTextColor` is red, then we get black text:

``` c++
r = 1.0 * 0; // 0
g = 0 * 1.0; // 0
b = 0 * 0; // 0
```

Long story short, to me that's more of a tint than a general color. Color could mean so many different things. Thoughts?
